c/include/libsbp/observation.h

```diff
@@ -612,6 +612,21 @@ coordinate system
 } msg_almanac_glo_t;
 
 
+/** GLONASS SV orbital and frequency slots mapping information
+ *
+ * The message reports mapping information regrding GLONASS SV orbital and
+ * frequency slots. Frequency numbers -7 to 6 have been reported as 1 to 14
+ * correspondingly. 0xFF for invalid. The 0th element and elements from 29th
+ * to 30th of the map are always invalid.
+ */
+#define SBP_MSG_FCNS_GLO             0x0072
+typedef struct __attribute__((packed)) {
+  u16 wn;      /**< GPS Week number [week] */
+  double tow;     /**< GPS Time of week [s] */
```

Comment:
Good point! I guess it should, as the ME should indeed have that piece of information when transmitting this message.. and a parser will benefit from placing this message in some time frame. However, if this introduces complexity (e.g., the message needing to be gated explicitly if time is not available), I would be in favor to either add a flag or remove the time. 
My reasoning was that indeed we do already have messages with a time tag like `MSG_GPS_TIME` and so handling could be similar.
One could also question to appropriateness of using a GPS time tag in a Glonass message.. but again in my reasoning was that being a "receiver time" rather than any specific constellation time.